_posts/2021-05-14-release-1.12.4.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title:  "Apache Flink 1.12.4 Released"
+date:   2021-05-14 00:00:00
+categories: news
+authors:
+- arvid:
+  name: "Arvid Heise"
+
+---
+
+The Apache Flink community released the next bugfix version of the Apache Flink 1.12 series.
+
+This release includes 73 fixes and minor improvements for Flink 1.12.3. The list below includes a detailed list of all fixes and improvements.
```

Comment:
73 seems a high number. I only find 21 Jira links in this blog post

Reply:
XD